src/main.cpp

```diff
@@ -358,28 +360,30 @@ void showUsage() {
          "Type 'man stenc' for more information.\n";
 }
 
+static void print_device_inquiry(std::ostream& os, const SCSI_PAGE_INQ *iresult)
+{
+  os << std::left << std::setw(25) << "Device Mfg:";
```

Comment:
`Mfg` = `Manufacturer`?

Reply:
How about `Manufacturer` or `Vendor` instead of `Device Mfg` to avoid abbreviations? There is limited space without changing a lot of other formatting, and I don't want to make PR an extensive change to behavior.

Reply:
sounds good.

Reply:
Changed to `Vendor` to most closely match the nomenclature in the SCSI specs.